src/libslic3r/PrintConfig.cpp

```diff
@@ -697,7 +697,7 @@ void PrintConfigDef::init_fff_params()
     def->label = L("Other layers");
     def->tooltip = L("Bed temperature for layers except the initial one. "
                      "A value of 0 means the filament does not support printing on the Cool Plate SuperTack.");
-    def->sidetext = "\u2103" /* °C */;	// degrees Celsius, don't need translation
+    def->sidetext = wxString::FromUTF8("\u2103") /* °C */;	// degrees Celsius, don't need translation
```

Comment:
I think these are unnecessary, because `sidetext` is `std::string` not `wxString`. Only those need `wxString`s need to explicitly set the encoding.

Reply:
Done.